tests/bdd/features/volume/resize/resize_online.feature

```diff
@@ -0,0 +1,93 @@
+Feature: Volume resize
+
+    Background:
+        Given a deployer cluster
+
+    Scenario Outline: Expand a volume
+        Given a published volume with <repl_count> replicas and all are healthy
+        When we issue a volume expand request
+        Then the request should succeed
+        And all the replicas of volume should be resized to new capacity
+        And the volume target should get resized to new capacity
+        And the volume should be expanded to the new capacity
+        And the new capacity should be available for the application
+        Examples:
+        | repl_count |
+        | 1          |
+        | 3          |
+
+
+    Scenario Outline: Expand a volume with an offline replica
+        Given a <publish_status> volume with more than one replicas
```

Comment:
Same thing here and further below as well, isn't unpublished already covered by offline?

Reply:
Segregated.